src/components/Modals/OnboardingReview.vue

```diff
@@ -0,0 +1,514 @@
+                  class="onboarding-label--review"
+                  >{{ getExamCredit(options) }}</label
+                >
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="onboarding-subHeader2-fillRow">
+          <span class="onboarding-subHeader2-review"> Transferred Course Credits</span>
+        </div>
+        <div class="onboarding-selectWrapper">
+          <div class="onboarding-selectWrapper-reviewExam">
+            <div>
+              <div v-for="(options, index) in displayOptions.class" :key="index">
+                <label v-if="options.class !== 'Select one'" class="onboarding-label--review">
+                  {{ options.class }}
+                </label>
+              </div>
+            </div>
+            <div class="alignEnd">
+              <div v-for="(options, index) in displayOptions.class" :key="index">
+                <label v-if="options.class !== 'Select one'" class="onboarding-label--review">
+                  {{ options.credits }} Credits
+                </label>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="onboarding-bottomWrapper">
+          <label class="onboarding-subHeader2-review">Total Transfer Credits:</label>
+          <div class="onboarding-label--bottom">
+            <label class="onboarding-label--bottom---bold">{{ totalCredits }}</label>
+            <label>Credits</label>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+// TODO: a lot of the functions are repeated from basic and transfer pages, is there a way to avoid that?
+
+import reqsData from '@/requirements/typed-requirement-json';
+import { examData } from '@/requirements/data/exams/ExamCredit';
+import coursesJSON from '../../assets/courses/courses.json';
+
+const placeholderText = 'Select one';
+
+export default {
+  props: {
+    user: Object,
+  },
+  data() {
+    // Set dropdown colleges and majors if already filled out
+    let collegeText = placeholderText;
+    let collegeAcronym = '';
+    let collegePlaceholderColor = '';
+    if (this.user.college !== '') {
+      collegeText = this.user.collegeFN;
+      collegeAcronym = this.user.college;
+      collegePlaceholderColor = '#757575';
+    }
+
+    let majorText = placeholderText;
+    let majorAcronym = '';
+    let majorPlaceholderColor = '';
+    if ('major' in this.user && this.user.major.length > 0) {
+      majorText = this.user.majorFN;
+      majorAcronym = this.user.major;
+      majorPlaceholderColor = '#757575';
+    }
+    let minorText = placeholderText;
+    let minorAcronym = '';
+    let minorPlaceholderColor = '';
+    if ('minor' in this.user && this.user.minor.length > 0) {
+      minorText = this.user.minorFN;
+      minorAcronym = this.user.minor;
+      minorPlaceholderColor = '#757575';
+    }
+    return {
+      // TODO: Get real college, major, and minor lists
```

Comment:
TODO?

Reply:
It was a TODO left by Ayesha, I have no idea what it means actually